src/lib.rs

```diff
@@ -238,17 +240,16 @@ extern crate log;
 extern crate native_tls;
 #[cfg(feature = "openssl-probe")]
 extern crate openssl_probe;
-#[cfg(all(feature = "native-tls", not(feature = "openssl")))]
-extern crate tokio_native_tls;
-#[cfg(feature = "webpki-roots")]
-extern crate webpki;
-#[cfg(feature = "webpki-roots")]
-extern crate webpki_roots;
-
 #[cfg(feature = "json-using-serde")]
 extern crate serde;
 #[cfg(feature = "json-using-serde")]
 extern crate serde_json;
+#[cfg(feature = "openssl")]
+extern crate tokio_native_tls;
+#[cfg(all(feature = "tokio-native-tls", not(feature = "openssl")))]
+extern crate tokio_native_tls;
```

Comment:
Can this be simplified as below?
```suggestion
#[cfg(any(feature = "tokio-native-tls", feature = "openssl"))]
extern crate tokio_native_tls;
```